expyfun/stimuli/_hrtf.py

```diff
@@ -151,3 +151,67 @@ def convolve_hrtf(data, fs, angle, source='barb'):
         brir = [resample(b, fs, brir_fs) for b in brir]
     out = np.array([np.convolve(data, brir[o]) for o in order])
     return out
+
+    
+def make_sym(x):
+    """
+    forces symmetry around zero
+    """
+    n = x.shape[-1]
+    x[..., :-((n + 1) // 2):-1] = np.conj(x[..., 1:(n + 1) // 2])
+    if np.mod(n, 2) == 0:
+        x[..., n // 2] = 0
+    return x
+    
+
+def interp_hrtf(angle, fs):
```

Comment:
I don't think we need a separate function for this. We can just add a parameter `interp=False` (by default) to our existing HRTF function. A user just does `interp=True` to explicitly say they are willing to use interpolation, then your code kicks in if the requested angle is not present.